dxr/plugins/clang/needles.py

```diff
@@ -1,28 +1,23 @@
-from operator import itemgetter
-from itertools import chain, izip
-from functools import partial
-
-from funcy import imap, is_mapping, repeat, icat
+from itertools import chain
 
 from dxr.indexers import (iterable_per_line, with_start_and_end,
-                          split_into_lines, Extent, Position)
+                          split_into_lines)
 
 
+# TODO: Use.
```

Comment:
It's not a test helper; it's a needle generator we never got around to building a filter for. We should do that at some point.